docs/developers/instantiate-ethermint.md

```diff
@@ -0,0 +1,54 @@
+# Setting Up Your Environment for Ethermint on Celestia
+
+Now the `ethermintd` binary is built, we need to setup a local network
+that communicates between `ethermintd` and Rollmint.
+
+## Run A Celestia Light Node
+
+All sovereign rollups need to submit their transaction data to
+Celestia.
+
+Here, we must first setup a Celestia Light Node with testnet tokens.
+
+You can do this by following this tutorial [here](./node-tutorial.mdx).
+
+## Instantiating the Ethermint Rollup
+
+With a Celestia Light Node running in one terminal session,
+we can proceed to generate the Ethermint rollup.
+
+In the `ethermint` directory, we have a helpful bash script that
+allows you to instantiate a local Ethermint sovereign rollup on Celestia.
+
+Run the following:
+
+```sh
+cd ethermint
+bash init.sh
+```
+
+This bash script does everything needed to initialize your Ethermint
+rollup.
+
+First, we need to setup some environment variables.
+
+<!-- markdownlint-disable MD013 -->
+```sh
+NAMESPACE_ID=$(echo $RANDOM | md5sum | head -c 15; echo;)
```

Comment:
```suggestion
NAMESPACE_ID=$(echo $RANDOM | md5sum | head -c 16; echo;)
```

Reply:
after error: `root@jcs:~/ethermint# ethermintd start --rollmint.aggregator true --rollmint.da_layer celestia --rollmint.da_config='{"base_url":"http://localhost:26659","timeout":60000000000,"gas_limit":6000000}' --rollmint.namespace_id $NAMESPACE_ID --rollmint.da_start_height $DA_BLOCK_HEIGHT
Error: invalid argument "c3bbfcc6e6e3a92" for "--rollmint.namespace_id" flag: encoding/hex: odd length hex string`